src/MainnetController.sol

```diff
@@ -253,6 +261,51 @@ contract MainnetController is AccessControl {
         );
     }
 
+    /**********************************************************************************************/
+    /*** Relayer Aave functions                                                                 ***/
+    /**********************************************************************************************/
+
+    function depositAave(address atoken, uint256 amount)
```

Comment:
`aToken`

Reply:
I would rather specify this as Aave to be clear at a high level.